lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -50,7 +50,7 @@
     <% mode_class = '' %>
   % endif
 <div class="course-container">
- % if course_program_info and course_program_info['category']=='xseries':
+ % if course_program_info and course_program_info.get('category')=='xseries':
```

Comment:
i'm fine with leaving it; missing `category` would mean there is definitely a data problem somewhere, but i'd rather the template hide the error from the user than blow up the whole page.